config/admins.txt

```diff
@@ -118,6 +118,7 @@ Aloraydrel = Game Master
 Quiltyquilty = Game Master
 SnipeDragon = Game Master
 Fjeld = Game Master
-kevinz000 = Game Master
+kevinz000 = Game Master //loser
+Tacolizard = Game Master //awesome dude
```

Comment:
As far as i know, config and data folders are not copied on the server, during autoupdate...